Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -158,12 +158,17 @@ class Mandelbrot : public GUI::Widget {
 
 void Mandelbrot::paint_event(GUI::PaintEvent& event)
 {
+    Frame::paint_event(event);
```

Comment:
You're calling the parent class' `paint_event` method twice (once before, once after). I've removed this one locally and it seems to make no difference. Is this intentional?

Reply:
Nope, unintentional. :^)